package.json

```diff
@@ -9,6 +9,7 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-scripts": "5.0.0",
+    "react-transition-group": "^4.4.2",
```

Comment:
Is the transition group not included as one of approaches for animation in React?
I can redo it later in case if this is not needed or not appropriate.

Reply:
Approved